Backpack-SwiftUI/NavigationBar/Classes/BPKNavigationView+Style.swift

```diff
@@ -54,7 +54,10 @@ public enum BPKNavigationBarStyle {
     }
     
     var verticalOffset: CGFloat {
-        safeAreasToIgnore.contains(.top) ? 0 : navigationBarHeight
```

Comment:
doing this calculation based on safe area was not correct and led to an issue in the transparent case!